xo_install.sh

```diff
@@ -1,24 +1,46 @@
 #!/bin/bash
-sudo apt-get install --yes nfs-common
+
+xo_branch="stable"
+xo_server="https://github.com/vatesfr/xo-server"
+xo_web="https://github.com/vatesfr/xo-web"
+n_repo="https://raw.githubusercontent.com/visionmedia/n/master/bin/n"
+yarn_repo="deb https://dl.yarnpkg.com/debian/ stable main"
+node_source="https://deb.nodesource.com/setup_5.x"
+yarn_gpg="https://dl.yarnpkg.com/debian/pubkey.gpg"
+n_location="/usr/local/bin/n"
+xo_server_dir="/opt/xo-server"
+xo_web_dir="/opt/xo-web"
+systemd_service_dir="/lib/systemd/system"
```

Comment:
/etc/systemd/system is usually just symlinks and .wants. Default is /lib/systemd/system.